adafruit_button/sprite_button.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-FileCopyrightText: 2022 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`adafruit_button.button`
+================================================================================
+
+Bitmap 3x3 Spritesheet based UI Button for displayio
+
+
+* Author(s): Tim Cocks
+
+Implementation Notes
+--------------------
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+from adafruit_imageload.tilegrid_inflator import inflate_tilegrid
+from adafruit_imageload import load
+from adafruit_button.button_base import ButtonBase
+
+
+class SpriteButton(ButtonBase):
+    """Helper class for creating 3x3 Bitmap Spritesheet UI buttons for ``displayio``.
+
+    :param x: The x position of the button.
+    :param y: The y position of the button.
+    :param width: The width of the button in tiles.
+    :param height: The height of the button in tiles.
+    :param label: The text that appears inside the button. Defaults to not displaying the label.
+    :param label_font: The button label font.
+    :param label_color: The color of the button label text. Defaults to 0x0.
+    :param string bmp_path: The path of the 3x3 spritesheet Bitmap file
+    :param string selected_bmp_path: The path of the 3x3 spritesheet Bitmap file to use when pressed
```

Comment:
`name` and `transparent_index` are not included in the documentation

Reply:
added both with the latest commit.